lib/utils/config/definitions.js

```diff
@@ -239,16 +240,24 @@ define('audit-level', {
 define('auth-type', {
   default: 'legacy',
   type: ['legacy', 'webauthn', 'sso', 'saml', 'oauth'],
-  deprecated: `
-    The SSO/SAML/OAuth methods are deprecated and will be removed in
-    a future version of npm in favor of web-based login.
-  `,
+  // deprecation in description rather than field, because not every value
+  // is deprecated
   description: `
-    What authentication strategy to use with \`adduser\`/\`login\`.
+    DEPRECATED: values "sso", "saml", and "oauth" will be removed in a future version.
```

Comment:
```suggestion
    NOTE: values "sso", "saml", and "oauth" will be removed in a future version.
```

The whole option isn't deprecated, having a `DEPRECATION` header makes it seem that way.  I think having the logging message is the most important, and then this entry can be less like the other deprecation messages.